tests/headbucket_test.py

```diff
@@ -0,0 +1,86 @@
+from unittest import *
+import boto3
+from boto3.s3.transfer import TransferConfig
+import botocore
+import botocore.session
+import inspect
+import os
+from libs.execute import *
+from libs.command import *
+from libs.utility import *
+from datetime import datetime
+
+class HeadBucket_Test(TestCase):
+
+    # ======== Construction, setUp, tearDown =========
+    rods_key = 's3_key1'
+    rods_secret_key = 's3_secret_key1'
+    alice_key = 's3_key2'
+    alice_secret_key = 's3_secret_key2'
+    s3_api_url = 'http://s3-api:8080'
+
+    def __init__(self, *args, **kwargs):
+        super(HeadBucket_Test, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+
+        self.boto3_client_rods = boto3.client('s3',
+                                        use_ssl=False,
+                                        endpoint_url=self.s3_api_url,
+                                        aws_access_key_id=self.rods_key,
+                                        aws_secret_access_key=self.rods_secret_key)
+
+        self.boto3_client_alice = boto3.client('s3',
+                                        use_ssl=False,
+                                        endpoint_url=self.s3_api_url,
+                                        aws_access_key_id=self.alice_key,
+                                        aws_secret_access_key=self.alice_secret_key)
+
+    def tearDown(self):
+        self.boto3_client_rods.close()
+        self.boto3_client_alice.close()
+
+    # ======== Tests =========
+
+    # Note:  Minio mc client does not have a head-object command but this is called behind the
+    # scenes on other tests such as list-object.
+
+    def test_botocore_head_bucket_rods(self):
```

Comment:
How do you feel about changing `_rods` to `_as_rods_user`?

Same for other tests.